Sources/LiveKit/Extensions/TimeInterval.swift

```diff
@@ -3,7 +3,8 @@ import Foundation
 /// Default timeout `TimeInterval`s used throughout the SDK.
 internal extension TimeInterval {
     static let captureStart: Self = 5
-    static let defaultConnect: Self = 10
+    static let defaultConnect: Self = 5
```

Comment:
Is this the connect timeout? We have seen that this is not enough on long delay networks? We are using 15 seconds in JS SDK for connection timeout.